en/docs/install-and-setup/setup/deployment-best-practices/production-deployment-guidelines.md

```diff
@@ -474,6 +474,18 @@ are reflected in the below mentioned config files, according to the new configur
 <p>See <a href="{{base_path}}/administer/product-configurations/default-product-ports/">Default Product Ports</a> for
  a list of common and product-specific ports used by WSO2 products.</p>
 </div>
+<div class="panelHeader" style="border-bottom-width: 1px;">
+<strong>Firewall between Traffic Manager and Gateway</strong>
+</div>
+<p>If you have a firewall between the Traffic Manager and the Gateway, you need to configure the heartbeat value to keep the JMS connection alive. To configure this open the <code>&lt;PRODUCT_HOME&gt;/repository/conf/advanced/qpid-config.xml</code> file and set the heartbeat value to a non zero value as shown below.</p>
+<div class="codeContent panelContent pdl">
+```
```

Comment:
``` 
is not valid when using HTML

Reply:
It is rendering HTML content properly. See the image below.
<img width="491" alt="firewall" src="https://user-images.githubusercontent.com/3957104/85549592-c73ac800-b63d-11ea-957f-72cd347af90c.png">

Reply:
<img width="998" alt="Screenshot 2020-06-24 at 17 13 23" src="https://user-images.githubusercontent.com/5195851/85549922-141e9e80-b63e-11ea-9a51-df12a5c35718.png">
I was looking at the rich diff and it showed as a formatting issue. Sorry for the noise.